package/MDAnalysis/coordinates/PDB.py

```diff
@@ -384,6 +384,8 @@ def _read_frame(self, frame):
             self.convert_pos_from_native(self.ts._unitcell[:3])
         self.ts.frame = frame
         self.ts.data['occupancy'] = occupancy
+        for aux in self.aux_list:
```

Comment:
As far as I can tell, `rewind()` for all the other readers works by going back before the first step and calling `next()`, which will update all the auxiliaries as well. PDB and XDR instead call `_read_frame(0)` which seeks in the file (and for XDR then calls `_read_next_timestep()`), so the auxiliaries don't go back to the first timestep with the trajectory. There's probably a nicer way to deal with that, but this seems to be working for them moment...